components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java

```diff
@@ -526,7 +545,7 @@ public List selectWorkerAggregatedThroughput(String workerId, long timeInterval,
      * @param query selection query.
      * @return the selected object.
      */
-    private List<List<Object>> selectGauge(String query) {
+    private List<List<Object>> selectGauge(String query,boolean isAggregated) {
```

Comment:
space after comma

Reply:
Fixed in d484a92d76bf096f387b03c2bc5b7945dead238b

Reply:
Move to separate repo.